src/style/disease-subpage.module.css

```diff
@@ -0,0 +1,86 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+    padding: 20px;
+    align-items: center;
+}
+
+.card, .diagram-card {
+    width: 75%;
+}
+
+.card :global(.ant-card-head),
+.diagram-card :global(.ant-card-head) {
+    background-color: var(--ALLEN_LIGHT_30);
+    color: var(--ALLEN_BLUE);
+    min-height: 64px;
+}
+
+.card :global(.ant-card-head-title),
+.diagram-card :global(.ant-card-head-title) {
+    /* allows long titles to wrap */
+    white-space: normal;
+    padding: 16px 0;
+}
+
+.card :global(.ant-card-head-wrapper),
+.diagram-card :global(.ant-card-head-wrapper) {
+    height: auto;
+}
+
+.card :global(.ant-card-body) {
+    padding: 16px;
+    border: 1px solid var(--ALLEN_BLUE);
+    border-radius: 5px;
+}
+
+.data-table {
+    padding: 0;
+    border: 1px solid var(--ALLEN_LIGHT_10);
+    border-radius: 4px;
+    margin-bottom: 10px;
+}
+
+.data-row {
+    padding: 17.5px 0;
+    border-bottom: 1px solid var(--OFF_WHITE);
+    padding-left: 20px;
+}
+
+.data-label {
+    font-weight: 400;
+    color: var(--MEDIUM_GRAY);
+}
+
+.legend-text {
+    font-size: 14px;
+    font-size: 16px;
+}
+
+.diagram-card {
+    margin-top: 24px;
+    border: 1px solid var(--ALLEN_BLUE);
+    position: relative;
+    padding-bottom: 40px;
+}
+
+.diagram-card-footer {
+    position: absolute;
```

Comment:
one small thing: multi-line captions are being cut off. I tried removing the fixed height so they can be wrapped, what do you think?